src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-logs.mdx

```diff
@@ -1,47 +1,59 @@
 ---
-title: 'OpenTelemetry logs: Best practices'
+title: OpenTelemetry logs in New Relic
```

Comment:
@jack-berg Since this page (along with metrics, resources, and traces pages) are no longer "best practices", I think we should upate the file path names. Perhaps something like `/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/data/logs`. Thoughts?

Reply:
I agree. Was holding off to reduce churn in case we revisit the navigation structure again. Why don't we see where things settle in the next week or two, and if we're comfortable with the navigation, update the path names to reflect it?